test/unit/puppetlabs/services/ca/certificate_authority_core_test.clj

```diff
@@ -455,6 +455,22 @@
         (is (true? (fs/exists? signed-cert-path)))
         (is (= 204 (:status response))))))
 
+  (testing "a signing request w/ a 'application/json' content-type and charset succeeds"
+    (let [settings         (testutils/ca-sandbox! cadir)
```

Comment:
speaking of principles, I believe that all of the alignment in these let bindings and maps is against the Linehan Principle.